src/modules/Builder/features/Activities/Activities.styles.ts

```diff
@@ -0,0 +1,10 @@
+import { styled } from '@mui/material';
+
+import { StyledBuilderBtn } from 'shared/styles';
+
+export const StyledAddActivityBtn = styled(StyledBuilderBtn)`
+  position: absolute;
+  top: 2.4rem;
+  right: 6.4rem;
+  z-index: 1100;
```

Comment:
pls use z-index variable from MUI

Reply:
done (button was moved to the ActivitiesHeader, excess styles were removed)